docs/samples/v1beta1/torchserve/model-archiver/README.md

```diff
@@ -0,0 +1,146 @@
+# Predict on a InferenceService using Torchserve
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/#install-kfserving).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+
+## Create PV and PVC
```

Comment:
I think we can make the high level steps a bit more clear and each can have a few sub steps
1) Create model store PV and PVC
 - Create PV
 - Create PVC
2) Create model store files layout and copy to PV
- Create pod for copying model store files to PV
- Create model store file layout on PV
- Copy model store files to the PV
- Delete the PV pod
3) Generate model archive file and server configuration file
- Create model archive pod and run model archive file generation script 
- Check the output and delete model archive pod